src/components/UserConsent/__tests__/index.test.tsx

```diff
@@ -24,6 +26,8 @@ const defaultOptions: StepComponentUserConsentProps = {
   trackScreen: jest.fn(),
 }
 
+console.error = jest.fn()
```

Comment:
Yeah just a quick fix to hide the unexpected `console.error` because of failed XMLHttpRequest. But I think @stefaniacardenas has fixed it in the NPM PR. Will merge later.